src/grandorgue/GOOrganController.cpp

```diff
@@ -1006,7 +1014,7 @@ GOMemoryPool &GOOrganController::GetMemoryPool() { return m_pool; }
 
 GOConfig &GOOrganController::GetSettings() { return m_config; }
 
-GOBitmapCache &GOOrganController::GetBitmapCache() { return m_bitmaps; }
+GOBitmapCache &GOOrganController::GetBitmapCache() { return *m_bitmaps; }
```

Comment:
As you anyway change this method, it could perhaps also be moved to the header file as the implementation is just one line?